### PR TITLE
Display all  books

### PR DESCRIPTION
**Pull request by nzhengs**
Since, I formatted the code using prettier, the code looks like all is changed. I will comment on the code I changed.

### CODE REVIEW

File: assets/scripts/app.js [hunk 1]
```diff
@@ -1,126 +1,156 @@
...
+  }
+  return {
+    init: init,
+    getBookList: getBookList
+  };
 })();
-getData.init(); 
+getData.init();
+
+const detailTemplate = `
```
Comment (nzhengs) [inline assets/scripts/app.js:129]: The code start here, it allows to display all the cards on the page.

File: index.html [hunk 2]
```diff
@@ -1,85 +1,78 @@
...
-                  </div>
-              </div>
-          </div>
-      </div>
-  </div>
-  <div>
-    <div class="container">
-      <div class="col s12 m7">
-        <div class="card horizontal">
-          <div class="card-image">
```
Comment (nzhengs) [inline index.html:60]: I have used this card as a multiline string in app.js

Comment (petr0n) [reply index.html:60]: This is awesome! Good thinking.

**Review by petr0n — APPROVED** [review]
Nice work Anjana

**Pull request merged**